src/main/java/ua/com/cinema/controllers/UserController.java

```diff
@@ -21,7 +22,7 @@ public UserController(UserService userService, UserMapper userMapper) {
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto getByEmail(@RequestParam String email) {
+    public UserResponseDto getByEmail(@RequestParam @Email String email) {
```

Comment:
does it work without the steps listed in this article?
https://www.baeldung.com/spring-validate-requestparam-pathvariable